CHANGELOG.md

```diff
@@ -4,23 +4,61 @@ This document outlines major changes between releases.
 
 ## [Unreleased]
 
+## [0.24.0] - 2022-09-14
+
 ### Added
-- Exposure of pool metrics (#615)
+
+- Exposure of pool metrics (#615, #680)
 - Configuration of `set_copies_number` (#634, #637)
 - Configuration of list of allowed `AccessKeyID` prefixes (#674)
+- Tagging directive for `CopyObject` (#666, #683)
+
 ### Changed
+
 - Improved wallet configuration via `.yaml` config and environment variables (#607)
-- Update go version for build to 1.19 (#694)
+- Update go version for build to 1.19 (#694, #705)
+- Update version calculation (#653, #697)
+- Optimized lock creation (#692)
+- Update way to configure `listen_domains` (#667)
+- Use `FilePath` instead of `FileName` for object keys (#657)
+
 ### Removed
+
+- Drop object search (#545)
```

Comment:
I think this point should be exposed in more detail. It's not clear where you dropped object search